### PR TITLE
added 3 main pages

### PR DESCRIPTION
**Pull request by TomHodson91**
3 main pages:

Home,
getStarted,
results

### CODE REVIEW

**Review by ohussein-0129 — DISMISSED** [review]
The structure is perfect. It is good to go. Inside the router a div is used and works as expected. Besides that, it is great.

**Comment by TomHodson91** [comment]
yes inside the router I've used a div as the router may only have one child

**Comment by ohussein-0129** [comment]
I am not sure if Ricardo needs to review it, you can merge it if you want as I would like to start creating the forms. I have approved it.

**Comment by rmallols** [comment]
let me have a look

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,21 @@
...
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
+        <Router>
```
Comment (rmallols) [inline src/App.js]: For simplicity reason, you can unwrap the `<Router>` to surround the `<div className='app'>`.

Hence, you wouldn't need to create the artificial `<div>` underneath to keep a single root element.

File: src/getStarted.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+
+export default function getStarted() {
+    return (
+        <div>
+            <Link to= {"/"}>
```
Comment (rmallols) [inline src/getStarted.js]: Mixing single and double quotes :)

**Review by rmallols — CHANGES_REQUESTED** [review]

**Comment by ohussein-0129** [comment]
Looks like you've already merged. It is better to put the div outside the router? I'll just wait for Ricardo's response.

**Pull request merged**